app/src/main/java/com/example/plantsapp/presentation/ui/worker/TasksWorkManager.kt

```diff
@@ -0,0 +1,57 @@
+// TODO remove this and maybe replace with multibinding
+@file:Suppress("WildcardImport", "NoWildcardImports")
+package com.example.plantsapp.presentation.ui.worker
+
+import android.content.Context
+import androidx.work.*
+import com.example.plantsapp.presentation.ui.notification.NotificationWorker
+import com.example.plantsapp.presentation.ui.utils.calculateDelay
+import java.util.Calendar
+import java.util.concurrent.TimeUnit
+
+class TasksWorkManager(appContext: Context) {
+    private val workManager = WorkManager.getInstance(appContext)
+
+    fun startNotificationWork(startDate: Calendar) {
+        workManager.enqueueUniquePeriodicWork(
+            NotificationWorker.NOTIFICATION_WORK_NAME,
+            ExistingPeriodicWorkPolicy.KEEP,
+            getEveryDayWorkRequest<NotificationWorker>(
+                startDate = startDate,
+                startHour = HOUR_OF_NOTIFICATION_WORK_STARTING
+            )
+        )
+    }
+
+    fun startReschedulingWork(startDate: Calendar) {
+        workManager.enqueueUniquePeriodicWork(
+            ReschedulingWorker.RESCHEDULING_WORK_NAME,
+            ExistingPeriodicWorkPolicy.KEEP,
+            getEveryDayWorkRequest<ReschedulingWorker>(
+                startDate = startDate,
+                startHour = HOUR_OF_RESCHEDULING_WORK_STARTING
+            )
+        )
+    }
+
```

Comment:
Maybe make these 2 methods private and encapsulate logic of starting this into one method? 
Like you've done with cancelAllWork